libs/application/template-api-modules/src/lib/modules/templates/new-primary-school/new-primary-school.utils.ts

```diff
@@ -349,13 +357,19 @@ export const transformApplicationToNewPrimarySchoolDTO = (
           nationalId: payerNationalId || '',
         },
       }),
-      childCircumstances: {
-        fieldInspection: fieldInspection === YES,
-        additionalDataProvisioning: additionalDataProvisioning === YES,
-        outsideSpecialist: outsideSpecialist === YES,
-        childViewOnApplication: childViewOnApplication === YES,
-      },
-      terms: terms === YES,
+      ...(isSpecialEducation &&
+        applicationType !== ApplicationType.CONTINUING_ENROLLMENT && {
+          childCircumstances: {
+            fieldInspection: fieldInspection === YES,
+            additionalDataProvisioning: additionalDataProvisioning === YES,
+            outsideSpecialist: outsideSpecialist === YES,
+            childViewOnApplication: childViewOnApplication === YES,
+          },
+        }),
```

Comment:
Shouldn't we use `shouldShowPage()` instead of `isSpecialEducation`? So it's the same condition used in childCircumstancesSubSection?

```suggestion
      ...(shouldShowPage(...) && {
          childCircumstances: {
            fieldInspection: fieldInspection === YES,
            additionalDataProvisioning: additionalDataProvisioning === YES,
            outsideSpecialist: outsideSpecialist === YES,
            childViewOnApplication: childViewOnApplication === YES,
          },
        }),
```